packages/salesforcedx-vscode-soql/CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# 50.14.0 - January 21, 2021
+
+## Added
+
+- SOQL code-completion on WHERE clause expressions ([PR #2839](https://github.com/forcedotcom/salesforcedx-vscode/pull/2839))
+- experimental option to perform remote query validation ([PR #2847](https://github.com/forcedotcom/salesforcedx-vscode/pull/2847))
```

Comment:
```suggestion
- Experimental option to perform remote query validation ([PR #2847](https://github.com/forcedotcom/salesforcedx-vscode/pull/2847))
```

Reply:
Do these entries get automatically picked up from PR description? Or are they manually copied/pasted? If automatic, we should discuss having some PR description guidelines to ensure LESS editing now. @praksb @smaddox-sf

Reply:
I am doing this manually for now since we are releasing separately and are not part of the extension pack. But I believe PDT has some automation around this. @sfsholden might know more.

Reply:
Our change log generator takes the squashed commit message. 

It cuts out the type (assuming there is one) from commitizen (this is used to determine if this should be an Add Fix or if it should be ignored), capitalizes the first word, and determines which section header to place it depending on what files were changed.